package.json

```diff
@@ -8,6 +8,9 @@
     "node": ">=12"
   },
   "scripts": {
+    "build": "npm run build:gitignore && npm run build:ts",
+    "build:ts": "tsc",
```

Comment:
will generated typings correctly merge with prepackaged knex ones?

Reply:
Intention for this is that for Knex 2.x the `.ts` file types are build time only and not included in the published npm package. Presumably a breaking change with Knex 3.x would be to remove the hand maintained `.d.ts` files and start including the ones from the compiler.